src/main/python/cfn_sphere/stack_configuration/parameter_resolver.py

```diff
@@ -89,49 +96,72 @@ def resolve_parameter_values(self, stack_name, stack_config, cli_parameters=None
         stack_outputs = self.cfn.get_stacks_outputs()
 
         for key, value in stack_config.parameters.items():
+            resolved_parameters[key] = self.resolve_parameter_value(key, value, stack_name, stack_config, stack_outputs)
 
-            if isinstance(value, (list, TransformList)):
```

Comment:
The diff from here down isn't really understandable, what's happening here?

Reply:
upstream have refactored and split `resolve_parameter_values` into `resolve_parameter_values` looping and callling `resolve_parameter_value`.